test/setup/index.ts

```diff
@@ -27,11 +26,9 @@ function getEnvVariableOrDefault(envVarName: string, defaultValue: string): stri
 export async function removeMonitor(): Promise<void> {
   await dumpLogs();
   try {
-    if (createCluster) {
-      await platforms[testPlatform].delete();
-    } else {
-      await platforms[testPlatform].clean();
-    }
+    await platforms[testPlatform].config();
```

Comment:
why did you add the `config` call here?

Reply:
Because of https://github.com/snyk/kubernetes-monitor/pull/395/files#diff-e5fa2adcbad65189e75788ac9a1fc12bR20-R28
At the start of the CI job/local test run, KUBECONFIG points to nothing